app/javascript/chat/chat.jsx

```diff
@@ -1309,15 +1305,6 @@ export default class Chat extends Component {
       return (
         <div className="chat__channels">
           {notificationsButton}
-          <button
-            className="chat__channelstogglebutt"
-            onClick={this.toggleExpand}
-            style={{ width: '100%' }}
-            type="button"
-            title="Expand channels"
-          >
-            {'>'}
-          </button>
           <Channels
             activeChannelId={state.activeChannelId}
             chatChannels={state.chatChannels}
```

Comment:
By removing the `if (state.expanded || !state.expanded)` statement above it will mean that this "fallback return" can be removed too